### DISCUSSION TITLE
Chromedriver 115+ Upwards not getting auto downloaded anymore now using Appium service in WebdriverIO

### DISCUSSION THREAD

**Discussion by RashmiShashindra** [webdriverio/webdriverio#11161] [Q&A]
Hi,

I am using the WebdriverIO Appium Service for my native mobile automation testing which also has webviews and context switching between the web and native.

With the new chromedriver 115 updates, All my tests are failing as It does not automatically download the chromedriver anymore.

**Below are my versions.**

```
"@wdio/appium-service": "^7.19.5",
"@wdio/cli": "^v7.20.1",
```

My appium config is as below (Appium Boilerplate configs)

```
import { config } from './wdio.shared.conf';

//
// ======
// Appium
// ======
//
config.services = (config.services ? config.services : []).concat([
  [
    'appium',
    {
      // This will use the globally installed version of Appium
      command: 'appium',
      args: {
        // This is needed to tell Appium that we can execute local ADB commands
        // and to automatically download the latest version of ChromeDriver
        relaxedSecurity: true,
        allowInsecure: ['chromedriver_autodownload'],
        address: 'localhost',
        // Write the Appium logs to a file in the root of the directory
        log: './appium.log',
      },
    },
  ],
]);
//
// =====================
// Server Configurations
// =====================
//
config.path = ['/wd/hub'];

export default config;
```

**Error I am getting:**

`unknown error: An unknown server-side error occurred while processing the command. Original error: No Chromedriver found that can automate Chrome '116.0.0'. See https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md for more details`

Appreciate your consideration on this.

Thanks in advance!

**Comment by christian-bromann** [comment]
> All my tests are failing as It does not automatically download the chromedriver anymore.

I've never heart that Appium would auto download Chromedriver. I recommend to post this issue in the Appium project if this worked before.

**Reply by RashmiShashindra** [reply]
@christian-bromann Thank you for the response. 

I have built my mobile code refering to the appium boilerplate example. In the configs it had the comment for this auto download the chrome driver. https://github.com/webdriverio/appium-boilerplate/blame/main/config/wdio.shared.local.appium.conf.ts 

Anyway I will post in the Appium project as you suggested and will update in this thread. :)
